api/rest/mantisbt_openapi.yaml

```diff
@@ -0,0 +1,1733 @@
+openapi: 3.0.0
+info:
+  title: Mantis Bug Tracker REST API
+  version: 2.27.0
+  license:
+    name: GPL-2.0
+    url: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+paths:
+  '/api/rest/filters/{filter_id}':
+    get:
+      summary: Get a filter
+      operationId: filterGet
+      description: ''
+      tags:
+        - 'Filters'
+      responses:
+        '200':
+          description: 'Success'
+        '4XX':
+          description: 'Bad Request'
+      parameters:
+        - name: filter_id
```

Comment:
Most parameters are reused all over the API, so they should be defined globally under *components* to avoid duplication